owncloudApp/src/main/res/layout/create_shortcut_dialog.xml

```diff
@@ -0,0 +1,115 @@
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:inputType="textNoSuggestions"
+                        android:textSize="15sp" />
+                </com.google.android.material.textfield.TextInputLayout>
+
+            </RelativeLayout>
+
+            <RelativeLayout
+                android:id="@+id/create_shortcut_dialog_url_section"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content">
+
+                <TextView
+                    android:id="@+id/create_shortcut_dialog_url_text"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/standard_half_padding"
+                    android:padding="@dimen/standard_padding"
+                    android:text="@string/create_shortcut_dialog_url"
+                    android:textColor="@color/black"
+                    android:gravity="bottom"
+                    android:textSize="15sp" />
+
+                <com.google.android.material.textfield.TextInputLayout
+                    android:id="@+id/create_shortcut_dialog_url_layout"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginEnd="@dimen/standard_half_padding"
+                    android:layout_toEndOf="@id/create_shortcut_dialog_url_text">
+
+                    <com.google.android.material.textfield.TextInputEditText
+                        android:id="@+id/create_shortcut_dialog_url_value"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:inputType="textNoSuggestions"
+                        android:textSize="15sp" />
+                </com.google.android.material.textfield.TextInputLayout>
+
+            </RelativeLayout>
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/standard_margin"
+                android:orientation="horizontal">
+
+                <androidx.appcompat.widget.AppCompatButton
+                    android:id="@+id/cancelButton"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="1"
+                    style="@style/Button.Borderless"
+                    android:text="@string/share_cancel_public_link_button" />
+
+                <androidx.appcompat.widget.AppCompatButton
+                    android:id="@+id/yesButton"
+                    android:enabled="false"
+                    android:textColor="@color/grey"
+                    style="?android:attr/borderlessButtonStyle"
```

Comment:
The style of this and the previous button is not the same, but they look the same. Let's unify it

Reply:
Done